simpleConditional/test_sortFourNumberAscending.py

```diff
@@ -0,0 +1,9 @@
+import unittest
+from sortFourNumberAscending import sortFourNumberAscending
+
+class TestSortFourNumberAscending(unittest.TestCase):
+    def test_listOfFourIntegerInput(self):
+        self.assertEqual(sortFourNumberAscending([5, 4, 3, 2]), [2, 3, 4, 5])
```

Comment:
Should not be in order like this. What if your program have bug and it just reverse your list?

Reply:
Acknowledge.